src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
enum members should be named in uppercase, and then if you need name in lowercase you can call it like this
`.name().toLowerCase()` 
Another option can be to add a field `value` for example and put there name in lowercase.